docs/repository/repository-github-apps.md

```diff
@@ -45,7 +45,7 @@ Once the GitHub Apps connection is suspended, KintoHub will lose access to the r
 6. Scroll down to the **Danger Zone** section
 7. Click on the **Suspend** button
 
-![Suspend](/img/anatomy/suspend.png)
+![Suspend](/img/anatomy/Suspend.png)
```

Comment:
action - change the name to be lowercase. Make sure all names of images are lower case + dash.

`suspend-me-please.png`